src/components/sbb-toast/sbb-toast.e2e.ts

```diff
@@ -82,6 +77,37 @@ describe('sbb-toast', () => {
     expect(element.getAttribute('data-state')).toEqual('closed');
   });
 
+  it('closes by marked action element', async () => {
+    page = await newE2EPage({
+      html: `
+      <sbb-toast>
+        <sbb-button slot="action" sbb-toast-close/>
```

Comment:
the sbb-toast-close attribute should be described in docs, not?

Reply:
It is described D: (I'll improve it a bit)
![image](https://github.com/lyne-design-system/lyne-components/assets/12052575/e748a0ee-e871-44bf-a811-ff8d6cc862d4)